.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
Sounds a bit off.
Maybe something like: "A clear and concise description of the problem you have experienced / encountered"

Reply:
I think it sounds fine. I've changed the example a tiny bit, but otherwise I see no issue with this wording since it's just an example.